tests/Http/Controllers/Api/Entries/DeleteControllerMediaLocalTest.php

```
@@ -0,0 +1,596 @@
+                    [
+                        "code" => "ec5_399",
+                        "title" => "Invalid project name",
+                        "source" => "deletion-entries"
+                    ]
+                ]
+            ]);
+        } catch (Throwable $e) {
+            $this->logTestError($e, $response);
+        }
+
+    }
```

PHPMD (2.15.0)

The method test_it_should_catch_wrong_project_name is not named in camelCase. (Controversial Rules)

(CamelCaseMethodName)